.github/actions/make/action.yaml

```diff
@@ -22,12 +22,16 @@ inputs:
     default: unknown
     required: false
 runs:
-  using: 'docker'
-  image: golang:1.17-bullseye
-  args:
-    - make
-    - ${{ inputs.target }}
-    - OUTPUT_DIR=${{ inputs.output-dir }}
-    - OS=${{ inputs.os }}
-    - ARCH=${{ inputs.arch }}
-    - VERSION=${{ inputs.version }}
+  using: 'composite'
+  steps:
+    # Cache go deps across builds
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.17'
```

Comment:
do we need a `cache: true` here to actually cache dependencies?

Reply:
Nope! Turns out if you do that it will cache between workflows, which we don't want. By default, it uses `actions/cache` (I think) under the hood to give the behavior we want (i.e. caching between jobs within the workflow). The proof is that the build jobs no longer have output showing that they are pulling Go deps.